File: aooc/basics/cpp/rev.cpp
#include<iostream>
using namespace std;

int main(){
    long long num = 0;
    cout<<"Enter Number : ";
    cin>>num;
    int rem = 0;
    long long rev = 0;
    while(num>0){
        rem = num%10;
        rev=(rev*10)+rem;
        num/=10;
    }
    cout<<"The reversed number is :"<<rev;
    return 0;
}

File: Practical/aooc/basics/rev.java
import java.util.Scanner;

public class rev{
	public static void main(String [] args){
		int num = 0;
		Scanner scen = new Scanner(System.in);
		System.out.print("Enter a large number : ");
		num = scen.nextInt();
		int rem =0;
		int rev = 0;
		while(num>0){
			rem=num%10;
			num/=10;
			rev= rev*10 + rem;
		}
		System.out.println(rev);
		scen.close();
	}
}
		

